### PR TITLE
Difference with excel results

### LINKED ISSUE #20: Last day of the year not being counted
https://github.com/kmedian/yearfrac/blob/0969623073de10a3273caaf19cbbb91fec726de6/yearfrac/act_afb.py#L22

Hi there, great library!
I believe we found a small bug.
The last day of the year is not being taken into account in the subtraction because 12/31 is the begging of the day(00:00:00hs)

We believe the answer could be just adding 1 day
` diffa = date_to_jd(y1, 12, 31) - date_to_jd(y1, m1, d1) + 1 `

Please tell me what you think, best regards

### PR DESCRIPTION
**Pull request by emiliobasualdo**
#20 
Great repo guys.
I was using it, an found some differences with Excel's results.

My commit is based on the idea that excel has the correct answer.
I am considering one more day basically because the last day of the year is not being taken into account in the subtractions.
A different approach would be something like this:
`diffa = date_to_jd(y2, 1, 1) - date_to_jd(y1, m1, d1) + 1`

Let me know what you think.
Regards

### CODE REVIEW

**Comment by emiliobasualdo** [comment]
@ulf1 Have you had a chance to take a look at it?
Regrads

**Comment by ulf1** [comment]
@emiliobasualdo Sorry. I'm busy with work rn. Please keep the PR open. I will take a look at it soon.

File: test/Excel-results.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from: 15/12/2018
+to: 1/3/2019
+
+basis   basis name          yearfrac(from, to, basis)
+0       US(NASD) 30/360     0.211111111111111
+1       Actual/actual       0.208219178082192
+2       Actual/360          0.211111111111111
```
Comment (ulf1) [inline test/Excel-results.txt:7]: yep

File: test/Excel-results.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+3       Actual/365          0.208219178082192
```
Comment (ulf1) [inline test/Excel-results.txt:8]: yep

File: test/Excel-results.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+4       European 30/360     0.211111111111111
```
Comment (ulf1) [inline test/Excel-results.txt:9]: yep

File: test/Excel-results.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from: 15/12/2018
+to: 1/3/2019
+
+basis   basis name          yearfrac(from, to, basis)
+0       US(NASD) 30/360     0.211111111111111
+1       Actual/actual       0.208219178082192
```
Comment (ulf1) [inline test/Excel-results.txt:6]: yep

File: test/test_act_afb.py [hunk 5]
```diff
@@ -6,7 +6,7 @@ class Test_ActAfb(unittest.TestCase):
 
     def test1(self):
         x = act_afb(2018, 12, 15, 2019, 3, 1)
-        self.assertEquals(round(x, 8), 0.20547945)
+        self.assertEquals(round(x, 15), 0.208219178082192)
```
Comment (ulf1) [inline test/test_act_afb.py:9]: yep

**Review by ulf1 — APPROVED** [review]
Thanks for finding the bug

**Pull request merged**